### ISSUE TITLE
Details on fine-grained NER model

### ISSUE THREAD

**Issue by mayhewsw** [allenai/allennlp-demo#109]
Can you say what data was used to train the fine-grained NER? I know it was Ontonotes v5. Did you train on train only? Or train+dev? Are there reported scores on Ontonotes test?

The model is found here:
https://github.com/allenai/allennlp-demo/blob/master/server/models.py#L49

The demo is here:
http://demo.allennlp.org/named-entity-recognition

Thanks!

**Comment by mayhewsw** [comment]
I'm guessing it's not trained on dev because the scores on Ontonotes dev are <<100

**Comment by DeNeutoy** [comment]
Hey Stephen,

I used the Ontonotes train set only to train this model. I don't actually have the numbers from the test set because I didn't run them - on the dev set it get's around 88.2 Avg F1.

Let me know if you have any more questions!

**Comment by SundeepPidugu** [comment]
Whats the link for the finegrained model ? to use with python.  Is this the correct fine grained model [Model](https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.12.18.tar.gz)

**Comment by schmmd** [comment]
@SundeepPidugu we now have usage information on the demo itself.

**Comment by vijayatgithub** [comment]
Hi , Please help me to find the latest pretrained - NER model released by Allen team?

Am using "fine-grained-ner-model-elmo-2018.12.21.tar" for name detection process but am noticing wrong results. I have used the same 'para content' in Allen Demo page there I could see expected results.

Looking forward your support to download the latest pretrained AllenNLP NER model.

**Comment by schmmd** [comment]
@vijayatgithub for any model you use you'll sometime get errors, and we have not improved this model significantly since we released it.

**Comment by vijayatgithub** [comment]
@schmmd - Thanks for your reply. Am passing the same input to demo page and offline version 
but result varies. 

I would like to detail my query further.

[chk1.txt](https://github.com/allenai/allennlp-demo/files/4774682/chk1.txt)
[chk2.txt](https://github.com/allenai/allennlp-demo/files/4774683/chk2.txt)
[chk3.txt](https://github.com/allenai/allennlp-demo/files/4774684/chk3.txt)
[chk4.txt](https://github.com/allenai/allennlp-demo/files/4774681/chk4.txt)

In the attachments, the offline model identifies person names correctly for the input files 'Chk1' and 'Chk2'.
The Chk3, Chk4 files also containing the same person names with one additional sentence, but the model returns "Others" instead of person tag.

Note: All 4 files working as expected in Demo page. 
https://demo.allennlp.org/named-entity-recognition

**Comment by schmmd** [comment]
@vijayatgithub I believe your dealing with https://github.com/allenai/allennlp/issues/3845.  You may need to warm up the model when you run it offline.  Please open a new issue if you have further questions.
